include/qemu/host-utils.h

```diff
@@ -142,6 +143,19 @@ static inline int clz64(uint64_t val)
     return val ? __builtin_clzll(val) : 64;
 }
 
+/**
+ * clz128 - count leading zeros in a 128-bit value.
+ * @val: The value to search
+ */
+static inline Int128 clz128(Int128 a)
```

Comment:
Yes, what i did here is actually incorrect since when CONFIG_INT128 is not defined Int128 is a struct, so just returning it causes a compilation error (and is overall unnecessary since clz128 max value is 128)